### ISSUE TITLE
[US - DEV] As MILES HARDIN, I want to accept the “terms and conditions” page so that I can submit my idea

### ISSUE THREAD

**Issue by JulieMass** [dev-launchers/dev-launchers-platform#750]

**Comment by JulieMass** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/IdeaSpace-Team-6350185d9843157427cc900b/issues/dev-launchers/dev-launchers-platform/750?planning-poker) with Zenhub @alicefate @dbradham @jordanmcousins @JulieMass @katehirschman @MariliaDP @Nouri-Anouar @rashikashaw @RETprojects

**Comment by JulieMass** [comment]
Hi @alicefate 

Thanks again for taking this ticket 🌟! 

As promised today I updated the information of the ticket with @MariliaDP link to prototype and @rashikashaw 's idea for the top nav behavior. Thank you all for everything. You are all amazing!

![image.png](https://images.zenhubusercontent.com/63400f3d1e79614e45d9ebd9/b00908c3-62e1-4f26-b469-1f469339fe39)

**Comment by alicefate** [comment]
@JulieMass Can you show it on screenshot, which should stay static, which should go away? 

And I don't think the page should have a  “I agree to the terms and conditions” button. It should be a file page, Miles could check on it when he submit or later. In my experience, no websites or apps have an agree button on this page.

**Comment by MariliaDP** [comment]
@alicefate You are right. If this is a file page that Miles can access anytime, it's better not to have an agree button. 
About the top nav behavior, I believe Dev Launchers' nav bar should go away when scrolling, and the back button + dropdown with links will remain static.
@JulieMass, let me know if you have a different opinion.

**Comment by MariliaDP** [comment]
Thanks for the update @JulieMass

**Comment by alicefate** [comment]
https://github.com/dev-launchers/dev-launchers-platform/assets/33647003/834c66b2-1110-4b67-b21a-1c5232509518

example on submission form

**Comment by JulieMass** [comment]
> tc.mp4 
> example on submission form

OMG Iris, this is amazing 👏👏👏! Thanks for sharing this video. The UI looks exactly like Marilia's designs and you even developed the stuck T&C at the top 😱! 

Question: are you still working on it? How does it look on smaller screens like mobiles? Are we sticking the back button and the dropdown for the T&C navigation links too?

![image](https://github.com/dev-launchers/dev-launchers-platform/assets/104226366/d449db52-fe13-4a33-82b7-61b3428b6e80)

**Comment by alicefate** [comment]
Thank you! I haven't fix my Internet issue, which means can't share screen on meeting. So I record this.

I got question for sticking part. 

Do all size screen has sticking or only smaller than 1280px(the size starting use right middle quick link)?

If all size having it, do all sticking should have back button or only the phone size has it?

For now, we have sticking quick link on screen size smaller than 1280px and all style is like what it shows in video. Because I think it looks more directly and reduce one click than drop-down. It's easy to change. What do you think?

**Comment by JulieMass** [comment]
> Thank you! I haven't fix my Internet issue, which means can't share screen on meeting. So I record this.
> 
> I got question for sticking part.
> 
> Do all size screen has sticking or only smaller than 1280px(the size starting use right middle quick link)?
> 
> If all size having it, do all sticking should have back button or only the phone size has it?
> 
> For now, we have sticking quick link on screen size smaller than 1280px and all style is like what it shows in video. Because I think it looks more directly and reduce one click than drop-down. It's easy to change. What do you think?

Thank you Iris! Replying to your question below:

> Do all size screens has sticking or only smaller than 1280px(the size starting use right middle quick link)? Yes, the sticky T&C nav behavior should only be applied to screens smaller than 1280px, which means that screens with two columns should not have it. The video you shared shows that the sticky behavior works perfectly for table sizes, but for smaller mobile screens, we should follow @MariliaDP's designs and have a dropdown T&C navigation with a back button.

![image](https://github.com/dev-launchers/dev-launchers-platform/assets/104226366/7aba4de7-ddc8-4c1d-ac16-9652f88d1f10)

**Comment by JulieMass** [comment]
@alicefate just passing by to let you know that I added what we decided in today's sprint review about the URL name in the acceptance criteria. 

![image.png](https://images.zenhubusercontent.com/63400f3d1e79614e45d9ebd9/e43c773c-f0cd-4d4d-b42a-b752f69cc7bd)

**Comment by dbradham** [comment]
@JulieMass @Rutuja177 the link to the terms and conditions from the idea submission form was never implemented as part of this ticket.

**Comment by JulieMass** [comment]
> @JulieMass @Rutuja177 the link to the terms and conditions from the idea submission form was never implemented as part of this ticket.

Thank you Dave. Well, that is unfortunate as it is part of the acceptance criteria, then it means the ticket isn't finished yet so we need to reassign it. @Rutuja177  would you be OK to finish developing what is pending from this ticket since it's related to the one you're working on for the PDF? Dave, what do you think about this? Let me know both your thoughts please,

**Comment by dbradham** [comment]
Sounds like a good plan to me! @JulieMass

**Comment by Rutuja177** [comment]
@dbradham @JulieMass Yes, I will start working on this. 

Thanks for the update.

**Comment by JulieMass** [comment]
> @dbradham @JulieMass Yes, I will start working on this.
> 
> Thanks for the update.

Hey @Rutuja177, thanks! I see you've already got ticket #1185 assigned to you in this sprint. So, do you think you'll be able to finish both that one and ticket #750 during this sprint? Or should we hold off on #750 and save it for a future sprint after you've completed your current assignments?
I'd rather you focus on finishing the tickets you've already got on your plate for this sprint, instead of taking on too much and not getting anything fully done by the end.

So, let me know before you start working on anything new, okay? If you can't get to #750 this sprint, we can just move it to the backlog and you can pick it up later. Just want to make sure we're on the same page!

Thank you.

(CC. @dbradham)

**Comment by Rutuja177** [comment]
Hey @JulieMass, to complete #1185, we need to start working on #750 to add the Terms and Conditions link. 
This is the design for Terms and Condition page attached above [Figma](https://www.figma.com/file/hEDIHsBZYa6hmK8As5mcjy/[DL---IdeaSpace]-UX-designs?type=design&node-id=2645-5947&mode=design&t=SGy3itS5gHqkom5o-0).

I think focusing on #750 before #1185, as without the Terms and Conditions link, we won't be able to add the PDF link. Regarding #750, I'm unsure if it will be finished in this sprint (by May 5th). We can either split it into two tickets to divide the tasks or move #750 to the next sprint as well. Let me know which option works best for you.

**Comment by JulieMass** [comment]
Moving this ticket to the **icebox** so Rutuja can focus on her current bug #1623. We can come back to this one later.

(CC. @Rutuja177 + @dbradham)
